deployment/src/main/java/io/quarkiverse/poi/deployment/POIProcessor.java

```diff
@@ -58,4 +58,13 @@ void registerXMLBeansClassesForReflection(CombinedIndexBuildItem combinedIndexBu
             reflectiveClass.produce(new ReflectiveClassBuildItem(false, true, implementor.name().toString()));
         }
     }
+
+    @BuildStep
+    public ReflectiveClassBuildItem registerLog4jClassesForReflection() {
+        return new ReflectiveClassBuildItem(true, true,
+                "org.apache.logging.log4j.message.ReusableMessageFactory",
+                "org.apache.logging.log4j.message.DefaultFlowMessageFactory",
+                "org.apache.logging.log4j.message.ParameterizedMessageFactory");
+    }
```

Comment:
For my understanding, are they from the API? Because you're not supposes to use the Log4j2 implementation so I'm surprised you need to add reflection there.